src/drivers/px4fmu/fmu.cpp

```diff
@@ -1026,74 +1026,75 @@ PX4FMU::cycle()
 			}
 		}
 
-		/* can we mix? */
-		if (_mixers != nullptr) {
+	}
 
-			size_t num_outputs;
+	_cycle_timestamp = hrt_absolute_time();
```

Comment:
The problem "Also, px4fmu quit running the output mixer when RC was lost." is due to the fact that the mixer was being run only on updates to the actuator topic(s). When commander asserts armed.force_failsafe, it appears that actuator values are no longer being published. How is this supposed to work?